### DISCUSSION TITLE
必當趴數

### DISCUSSION THREAD

**Discussion by Yanxxx7979** [1101-GS4719-C/Python-Community-GS4719-C#1892] [General]
https://udn.com/news/story/6928/7688077
大家是否都有過被當危機!
我對於被當的意義 理解是因為對於這堂課的內容不夠熟練所以希望學生能夠再修一次(好課值得一修再修:)
但曾聽教授在課上說 系上給他壓力要求必須當1/4得學生等等
而老師也反應 覺得這樣的做法不是恨恰當，學生都已經考及格還要當他 不合理
我也覺得若是大家對於科目都有達到應有的熟練度(可能是及格的標準)，這樣的話還一定要當到固定比例的人嗎!

**Comment by BeaGle0928** [comment]
我覺得如果當人是一定要當到一定的趴數的話那麼分數好像就沒有太大存在的意義，因為不存在所謂60分的及格線。或許對於同學間的競爭上能夠有所提升，但對於一些同學，這個科目就是真的讀不懂，好不容易讀到了六十幾分，結果還是因為趴數的關係被當，那樣的話一定很會傷心。

**Comment by vchien03** [comment]
沒想到有這種「規定」耶...說實話我覺得這種做法不太公平，總是會有一些人不擅長拿到漂亮的分數，但那不一定代表他不認真或對該科目不熟悉，我覺得用這種"後1/4一定會被當"的方式不太妥當。

**Comment by benbebobo149** [comment]
我認為或許可以換個角度思考，當掉那麼多人對老師來說究竟有沒有好處呢？我認為弊大於利。首先，學生可能會對老師提出投訴，或在教師評鑑上給予低分，甚至有些學生可能會找記者報導事件，使情況更為惡化。我甚至想不出一個老師當掉那麼多人會帶來什麼好處，除非他喜歡虐待其他人吧，哈哈。這讓我反思到了對待他人的方式和處事方法。最後，感謝同學的分享，讓我有機會思考這個問題。
https://news.ltn.com.tw/news/life/breakingnews/4561274
這個連結就是我說的那個例子，有興趣的朋友可以看一下。

**Comment by benbebobo149** [comment]
我覺得教授這樣說只是在嚇唬人，提醒大家要努力讀書，不要鬆懈。但如果一直講這種話，可能會像放羊的小孩一樣，到最後沒有一個學生相信老師會真的如此行動。然而，這種情況其實並非不可能。有時候，我們可能會低估自己的能力，以為自己永遠不會成為統計數據中的一部分。但事實上，或許有一天，他真的成為了那樣的比例，並不是他故意要這樣，而是真的大家都表現得不理想。這讓我反思了教育的方式和言教與行為之間的差距。或許我們應該更多地關注行動，而不僅僅是口頭上的勸告。最後，感謝同學的分享，讓我有機會思考這個議題，並從中獲得啟發。

**Comment by EPHMR0524** [comment]
這種固定當一定比例的人規則我只在我爸媽那個年代的私立學校(中原)會有這樣的規則，我覺得以中央學生的努力程度與課業難度，這麼做明顯是不公平的，原始成績能過的學生基本上都已經是有一定能力的，沒有必要硬是為了當一定比例的人而將分數調低使固定比例的學生被當。

**Comment by 20050201Shawn** [comment]
這樣的規則顯然對於原始成績就及格卻被必當率搞的同學不公平，若是怕課程過於簡單或沒有競爭性，教授可以選擇增加課程難度讓過的人不要那麼多，但同時讓學生可以學到東西，而不是為了當人而當人，導致弊大於利。若是真的單純喜歡當人的話，那教授比較適合轉行做折磨人的職業，例如牙醫。

**Comment by lyykey** [comment]
曾經遇到極度狠角色老師，第一堂課說往年會當掉1/3的學生，看了前左右同學，我就在想應該被當的是他們兩個哈哈。
雖然老師講這些警告話，勸退不少人不要修這堂課，但我發現講這些話確實可以為大家帶來積極感，怕自己就是被當的那個。
還好我過了~

**Comment by YANRONG0123** [comment]
我曾聽聞過某門課要修兩次，才能夠比較好理解，但是被當掉的話，在成績單上就是不太好看，所以還是會熬夜把它拚過去，過與不過還是和時間分配比較有關係，雖然有些課真的比較硬，但很多人明明知道這個狀況，結果還在整天吃喝玩樂，最後還是被當掉。

**Comment by YuanWang08** [comment]
我覺得這也是現在教育體制的問題，對於該怎麼授予學分沒有一個統一的標準。有些教授不在意出席只用考試評量，有些只要有出席就幾乎不太當人。如果只要出席就能獲得學分，那讀大學似乎也就像是在坐牢荒廢人生。我個人認為最好的方式是舉行考試，考過了才能獲得至少六十分以上的分數。而且在每學期初也會先舉辦一次，像有些學校會有微積分的檢定考，只要通過了就可以免修，我希望中央未來也能推行這種制度，讓先備知識較好的學生不用浪費一學期的時間坐在教室聽著已經會的東西。不是說教授上課沒用，只是同樣的時間下，把時間留給學生的效益對於學生比較高。

**Comment by William08290829** [comment]
我覺得這種制度如果是放在必修課對某些人來說真的是蠻不公平的，畢竟有些人已經很認真但還是考不到那樣的分數來說，這根本是不讓他畢業。但如果是一般的選修或通識課，教授如果要這樣做的話，那至少那些努力但過不了的人大不了下次不要選就好了

**Comment by yxin214** [comment]
我覺得這個制度滿不合理的，如此大家為了不被當掉必須不成為落後的那25%，或許有助於整體成績的進步，但我認為沒必要。若有及格代表學生對此科目已有一定的了解也努力於該科目，我們應將時間投入於更有興趣的科目，而不是為了不被當掉花費過多的時間在沒興趣的科目上。

**Comment by cassy0201** [comment]
我覺得如果自己遇上這種制度會蠻生氣的,如果我已經及格代表我有一定的能力，只是興趣可能沒有那麼多，為什麼還要再修一次一樣的東西浪費時間呢

**Comment by 155Julia** [comment]
作為一個學生，我對這種必須當掉固定比例學生的做法感到困惑和不解。學習的本質是掌握知識和技能，而不是為了符合某種比例而讓部分學生被當。如果每個學生都能達到課程要求，我認為應該鼓勵和支持他們，而不是因為制度上的原因而故意刁難。這樣做不僅打擊學生的學習積極性，還可能讓一些有潛力的學生因為一次失誤而受到不必要的挫折。我認為學校和老師應該更多地考慮學生的實際情況，制定更科學合理的評分標準，真正做到因材施教。這樣才能真正提高教育質量，讓每個學生都能充分發揮他們的潛力。

**Comment by YeeDaniel** [comment]
這個我個人非常有感，當初大一上的企概因為期中考巨爛，期末考及格邊緣，還好老師有出加分作業，讓我剛好過了，這邊真的謝謝周老師！下學期還有周老師的課，這次一定認真上課了QQ，不敢耍廢了

**Comment by yubo0oo** [comment]
通過科目是學生的本分
但如果老師是固定抓最後百分之20當掉之類的
個人認為相當過分
